tests/unit/test_control_bar.py

```diff
@@ -0,0 +1,18 @@
+from callbacks.control_bar import reset_filters
+import pytest
+import numpy as np
+
+
+class TiledDataMock:
+    def __init__(self):
+        self["sample_project"] = np.zeros(500, 500, 500)
```

Comment:
```suggestion
@pytest.fixture
def tiled_data_mock():
    return {"sample_project": np.zeros((500, 500, 500)), "reconstruction": 0}
```

Reply:
@Farkites could you explain why you would approach this as a fixture instead of mocked class?